src/wallet/walletdb.cpp

```diff
@@ -524,7 +524,7 @@ DBErrors CWalletDB::LoadWallet(CWallet* pwallet)
     bool fNoncriticalErrors = false;
     DBErrors result = DB_LOAD_OK;
 
-    LOCK(pwallet->cs_wallet);
+    AssertLockHeld(pwallet->cs_wallet);
```

Comment:
This will no longer compile due to the missing lock annotation in the header.

Could just remove it for now?